specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -692,78 +692,78 @@ model AcsRouterWorkerSelector {
   key?: string;
 
   /** Router Job Worker Selector Label Operator */
-  operator?: AcsRouterLabelOperator;
+  operator: AcsRouterLabelOperator;
 
   /** Router Job Worker Selector Value */
   #suppress "@azure-tools/typespec-azure-core/no-unknown" "The type of labelValue is object"
-  labelValue?: unknown;
+  labelValue: unknown;
 
   /** Router Job Worker Selector Time to Live in Seconds */
   @encode("ISO8601") // TODO verify this is timespan in C#
-  ttlSeconds?: duration;
+  ttlSeconds: duration;
```

Comment:
There is something odd going on here. I have an older version of the JSON for this that lists 'ttlSeconds' as a float, not a string (or duration).

Looking at the swagger I see it's the case there as well: https://github.com/Azure/azure-rest-api-specs/blob/44cbf51b9eee8a2dbf55ced61a61032aaed0148b/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json#L2167

HOWEVER, looking at the user-facing docs, they've doc'd it as a string! So I'm not 100% on what is right here - did they make a breaking change at some point?

https://learn.microsoft.com/en-us/azure/event-grid/communication-services-router-events#microsoftcommunicationrouterjobqueued

CC: @lmazuel